src/main/java/com/project/model/PasswordResetToken.java

```diff
@@ -0,0 +1,34 @@
+package com.project.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Data
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+public class PasswordResetToken {
+
+    private static final int EXPIRATION = 60 * 24;
```

Comment:
This should not be exposed here. Please add to properties file like you did with JWT information.